src/main/java/io/kuzzle/sdk/core/KuzzleDataCollection.java

```diff
@@ -70,8 +70,8 @@ public KuzzleDataCollection(@NonNull final Kuzzle kuzzle, @NonNull final String
    * @param filter   the filter
    * @param listener the listener
    */
-  public void advancedSearch(final JSONObject filter, final KuzzleResponseListener<KuzzleDocumentList> listener) {
-    this.advancedSearch(filter, null, listener);
+  public void search(final JSONObject filter, final KuzzleResponseListener<KuzzleDocumentList> listener) {
```

Comment:
If I remember right, we decided to keep `advancedSearch`, printing a one-time message on standard output saying that it's deprecated and advising to use `search` instead, and then transparently calling the new `search` method.
And we settled to remove `advancedSearch` completely for Kuzzle v2.